config/initializers/cookies_serializer.rb

```diff
@@ -2,4 +2,4 @@
 
 # Specify a serializer for the signed and encrypted cookie jars.
 # Valid options are :json, :marshal, and :hybrid.
-Rails.application.config.action_dispatch.cookies_serializer = :marshal
+Rails.application.config.action_dispatch.cookies_serializer = :hybrid
```

Comment:
This is a false positive. The cookies are signed and will not be deserialized without a valid signature. The same warning occurs with `:marshal` which was the previous setting: https://github.com/presidentbeef/brakeman/blob/391f530d6d685455fa4a5a8c9f5d29d6bc03af73/lib/brakeman/checks/check_cookie_serialization.rb#L12.

This was [discussed](https://github.com/presidentbeef/brakeman/discussions/1710#discussioncomment-2801601) in the Brakeman project:
> Crafting a malicious session cookie does require the attacker to know the secret key ...

This was set to `:hybrid` rather than `:json` to avoid breaking existing "remember me" cookies.

Reply:
Will this change in the future, should we have a todo ?

Reply:
It would be a good idea to change the format to `:json` once all current "remember me" tokens have expired. That will be 3 months after this change is deployed (expiration of `remember_user_token` cookie).